config/hyper_kjm.yml

```diff
@@ -0,0 +1,71 @@
+log: True
+save_path: "/glade/work/kjmayer/research/ASD/s2sml/results/"
+
+# hello there
+
+pbs:
+  jobs: 10
+  kernel: "ncar_pylib /glade/work/$USER/py37"
+  bash: ["source ~/.bashrc; module load ncar_pylib"]
+  batch:
+    N: "ms-2250"
+    l: ["select=1:ncpus=8:ngpus=1:mem=128GB", "walltime=12:00:00", "gpu_type=v100"]
+    A: "NAML0001"
+    q: "casper"
+    o: &alias "/glade/work/kjmayer/research/ASD/s2sml/results/out"
+    e: *alias
+    
+optuna:
+  storage: "s2s.db"
+  study_name: "vanilla"
+  storage_type: "sqlite"
+  objective: "/glade/work/kjmayer/research/ASD/s2sml/applications/train.py"
+  direction: "minimize"
+  metric: "valid_mae"
+  n_trials: 1000
+  gpu: True
+  sampler:
+    type: "TPESampler"
+    n_startup_trials: 100
+  pruner:
+    type: "MedianPruner"
+    n_startup_trials: 50
+  parameters:
+    optimizer:learning_rate:
+      type: "loguniform"
+      settings:
+        name: "learning_rate"
+        low: 1.0e-06
+        high: 1.0e-02
+    data:norm:
+      type: "categorical"
+      settings:
+        name: "norm"
+        choices: ["None", "minmax", "negone", "zscore"]
+    trainer:weight_decay:
+      type: "loguniform"
+      settings:
+        name: "weight_decay"
+        low: 1.0e-12
+        high: 1.0e-03
+    trainer:train_batch_size:
+      type: "int"
+      settings:
+        name: "train_batch_size"
+        low: 4
+        high: 256
+    trainer:training_loss:
+      type: "categorical"
+      settings:
+        name: "training_loss"
+        choices: ["mae", "mse", "smooth", "logcosh", "xtanh", "xsigmoid", "huber"]
+    model:name:
+      type: "categorical"
+      settings:
+        name: "model_name"
+        choices: ["unet", "unet++", "manet", "linknet", "fpn", "pspnet", "pan", "deeplabv3", "deeplabv3+"]
```

Comment:
Let's remove the "pan" model from the list of choices -- it doesn't work with our data